README.md

```diff
@@ -20,3 +20,83 @@ var.DEPLOY_NAME
 var.prefix
   Enter a value: example-cumulus-dev
 ```
+
+## Usage Example
+To use the task, add it as the first step in your ingest workflow followed by
+SyncGranule.
```

Comment:
should we maybe call out the sync granule docs? Just as a nice thing to do?

Reply:
Normally, that would seem like a nice idea to me, however, our recent experience with those docs makes me think it might actually by more hurtful than helpful to send people there. So... Yea idk.